src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,21 @@
+import Card from 'react-bootstrap/Card';
+import './ArticleCard.css'
+
+export default function ArticleCard(props) {
+    const { article } = props
+
```

Comment:
Please include a loading pattern to improve the user experience while they’re waiting for the page to render.

Reply:
Done